hnn_qt5.py

```diff
@@ -218,71 +231,174 @@ def runsim (self):
     print(''); self.updatewaitsimwin('')
   
   def optmodel (self):
+    import simdat
+    simdat.initial_ddat = simdat.ddat.copy()
```

Comment:
Stores the initial fit (from "Run Simulation"). To be displayed in final dipole plot as black dashed line.

Reply:
good idea